matlab/CMakeLists.txt

```diff
@@ -32,8 +110,29 @@ endif()
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake_modules)
 
-# Arrow is Required
-find_package(Arrow REQUIRED)
+# Only build the MATLAB interface C++ tests if MATLAB_BUILD_TESTS=ON.
+if(MATLAB_BUILD_TESTS)
+  # Should support custom GTEST_ROOT as well as package managers.
```

Comment:
`-DGTest_ROOT` should work with only `find_package(GTest)`. We don't need additional code for it: https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure

Reply:
My apologies - I think the wording of my comment on line 115 may have lead to some confusion. What I meant by this comment was "`find_package(GTest) should already automatically support GTEST_ROOT and package managers`, not that we "should add more code to enable this".

I'll reword this comment to be less ambiguous.

Reply:
Fixed.

Reply:
Ah, sorry for my misunderstanding.

Reply:
No need to apologize! The original wording was confusing.